lib/shared/bucketing-assembly-script/assembly/types/configBody.ts

```diff
@@ -12,24 +12,22 @@ export class PublicProject extends JSON.Value {
     _id: string
     key: string
     a0_organization: string
-    settings: JSON.Obj | null
+    settings: JSON.Obj
 
     constructor(project: JSON.Obj) {
         super()
         this._id = getStringFromJSON(project, '_id')
         this.key = getStringFromJSON(project, 'key')
         this.a0_organization = getStringFromJSON(project, 'a0_organization')
-        this.settings = getJSONObjFromJSONOptional(project, 'settings')
+        this.settings = getJSONObjFromJSON(project, 'settings')
```

Comment:
does every config in the CDN have a settings field now?

Reply:
https://github.com/DevCycleHQ/js-sdks/blob/main/lib/shared/types/src/types/config/models/project.ts#L12 we made it required here already and treat it as required in other places, so I think so